assets/app/views/project.html

```diff
@@ -80,9 +80,9 @@ <h2 class="service" ng-if="!displayRouteByService[serviceId]">
                 Show only the first two ports if there are many since we don't have much space here.
                 The full list is visible elsewhere.
                 -->
-                <span ng-repeat="portMapping in service.spec.ports | orderBy:'port' | limitTo:2" class="nowrap">
```

Comment:
Why move nowrap inside? This parent element already gets repeated once for each port, and we don't want it to wrap internally...